product_price_inquiry/product_inquiry_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data> 
+	<record id="view_inquiry_form" model="ir.ui.view">
+        <field name="name">price.inquiry.form</field>
+        <field name="model">product.inquiry</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form string="Sales Order">
+                <field name="product_id" colspan="2" required ="1"/>
+				<field name="qty" colspan="2"/>
+                <field name="pricelist_id" colspan="2"/>
+                <field name="price_unit" invisible="1"/>
+				<button name="action_button_inquiry"  colspan="2" string="Inquiry" type="object" icon="gtk-execute"/>
+                <field colspan="4" name="inquiry_line" nolabel="1">
+                    <tree string="Product Price Inquiry Line">
+                        <field name="pricelist_id"/>
+                        <field name="product_id"/>
+                        <field name="qty"/>
+                        <field name="price_unit"/>
+                    </tree>
+			 	</field>
+            </form>
```

Comment:
@gurneyalex  Thanks! maybe because of the mix of space and indent. I'll fix that by all using space.